crates/sui-core/src/checkpoints/mod.rs

```diff
@@ -686,15 +684,16 @@ impl CheckpointBuilder {
                 );
                 self.metrics.highest_accumulated_epoch.set(epoch as i64);
 
-                // for now, just log this value, and insert default val into checkpoint summary
+                // for now, just log this value. Later it must be included in
+                // EndOfEpochData::epoch_commitments
                 info!("Epoch {epoch} root state hash digest: {root_state_digest:?}");
 
                 Some(EndOfEpochData {
                     next_epoch_committee: committee.voting_rights,
                     next_epoch_protocol_version: ProtocolVersion::new(
                         system_state_obj.protocol_version(),
                     ),
-                    root_state_digest: Accumulator::default().digest(),
+                    epoch_commitments: vec![],
```

Comment:
Can we explain a bit why this is empty? (at least add a comment)

Reply:
Sure - we just haven't enabled this yet because the feature is still being debugged.